src/main/java/org/opensearch/flowframework/model/State.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.flowframework.model;
+
+/**
+ * Enum relating to the state of a workflow
+ */
+public enum State {
```

Comment:
```suggestion
public enum WorkflowState {
```

Reply:
I already have a WorkflowState file in the same package here for the actual workflowstate object.